Bottom: 947c8f3fcc13efe3e71abf0b6444f5b8ac71fef4
Top:    8be05b8bf7dc50eb3740ea9bea34652e119246ed
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 23:58:57 +0000

Refresh of debug-stuff

---

diff --git a/src/hid/step/step.c b/src/hid/step/step.c
index 52182c0..47499ce 100644
--- a/src/hid/step/step.c
+++ b/src/hid/step/step.c
@@ -152,11 +152,7 @@ step_do_export (HID_Attr_Val * options)
   if (filename == NULL)
     filename = "pcb-out.step";
 
-#if 1
   board_outline_list = object3d_from_board_outline ();
-#else
-  board_outline_list = NULL;
-#endif
 
   board_outline = board_outline_poly (true);
   piece = board_outline;
